_partials/_migrate_live_migration_docker_subcommand.md

```diff
@@ -69,7 +69,37 @@ Press 'c' and ENTER to continue
 
 ### 3.b Perform live-migration
 
-Next, open a new terminal and initiate the live migration. Allow it to
+The `migrate` subcommand supports following flags
+
+```sh
+docker run --rm -it --name live-migration-migrate \
+    -e PGCOPYDB_SOURCE_PGURI=$SOURCE \
+    -e PGCOPYDB_TARGET_PGURI=$TARGET \
+    --pid=host \
+    -v ~/live-migration:/opt/timescale/ts_cdc \
+    timescale/live-migration:v0.0.16 migrate --help
+usage: main.py migrate [-h] [--dir DIR] [--resume] [--skip-roles] [--table-jobs TABLE_JOBS] [--index-jobs INDEX_JOBS]
+                       [--skip-extensions [SKIP_EXTENSIONS ...]] [--skip-table-data SKIP_TABLE_DATA [SKIP_TABLE_DATA ...]]
+
+options:
+  -h, --help            Show this help message and exit
+  --dir DIR             Working directory
```

Comment:
Can we just remove this flag? It is not useful and might give false impression that they could use different directory to store migration data and the customer might endup in trouble.